### ISSUE TITLE
Whitelist an element in cosmetic filters list for Croatian education portal

### ISSUE THREAD

**Issue by Mel34** [uBlockOrigin/uAssets#6304]
### URL(s) where the issue occurs
`https://e-dnevnik.skole.hr/`
^ This is the official schooling system portal in Croatia.

### Describe the issue

With cosmetic blocking rules enabled on this domain subject picker box is hidden which breaks the functionality of the site.
Cosmetic filters block the  subject selection box originating at `https://e-dnevnik.skole.hr/grade_book/course_activities`. As a result subjects teachers teach cannot be selected.

### Screenshot(s)

I'd rather not post screenshots because I'd like to remain anonymous. I will provide them privately to the repo owner if _really_ needed.

### Versions

- Browser/version: firefox 69
- uBlock Origin version: v1.22.2
*note: issue isn't present on chromium*

### Settings

Irrelevant to this issue, it's the inbuilt cosmetic filter that intercepts this one.

### Notes

TL:DR disabling cosmetic filtering or adding `https://e-dnevnik.skole.hr/grade_book/course_activities` to whitelist unbreaks some school portal website.

**Comment by mapx-** [comment]
- Are you using some regional filter list ?
- Does this filter work ?

`@@||e-dnevnik.skole.hr^$generichide`

- if does not work: open uBo logger, refresh the page, select all the blocked items / requests and post them here (as screenshot eventually)

**Comment by Mel34** [comment]
No, not using any regional filter lists.
That filter did work, I've removed the whitelist first.

**Comment by mapx-** [comment]
ok, I'll add it to `unbreak list`

**Comment by Mel34** [comment]
Much appreciated!
